doc/cephadm/install.rst

```diff
@@ -6,6 +6,12 @@ Cephadm creates a new Ceph cluster by "bootstrapping" on a single
 host, expanding the cluster to encompass any additional hosts, and
 then deploying the needed services.
 
+Each host should be prepared as follows for deployment:
+
+- Allow SSH for the root user
+- For the first "bootstrapping" host and any other hosts intended to be monitors, set up passwordless SSH to all other hosts
```

Comment:
This is not required. cephadm uses it's own ssh config and its own certificate to connect to other hosts. As long as you make sure the certificate works, users should be fine.

Reply:
Maybe I am misunderstanding the "add hosts to the cluster" section then.  Is the intention that the ceph.pub generated on the bootstrap node is transferred to each new host, and then copied into the authorized keys file on that host?  If so, I can update this section to make it clear that step 1 is run on hosts and step 2 is run on the bootstrap node.

Reply:
If both steps are intended to be run on the bootstrap node, then running ssh-copy-id with root as the user prompts for a password, and will fail if root SSH is not enabled.

Reply:
Apparently my certificate doesn't work, and it is not clear how to make sure it works, because there is no information about what is expected.  I can update it if anyone can share what is expected.

Reply:
Maybe this helps?

https://github.com/ceph/ceph/blob/71207fc834a3ec994b230c418d8895084d02bebd/src/pybind/mgr/cephadm/module.py#L1134-L1143